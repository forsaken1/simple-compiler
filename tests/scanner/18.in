5.435E-2