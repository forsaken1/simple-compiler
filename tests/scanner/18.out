1	1	float		5.435E-2
1	9	eof		End of file
